sdk/python/tests/utils/cli_utils.py

```diff
@@ -76,6 +76,9 @@ def local_repo(self, example_repo_py: str, offline_store: str):
                 path: {data_path / "online_store.db"}
             offline_store:
                 type: {offline_store}
+            flags:
```

Comment:
any particular reason for this change? I think we usually accomplish this with `IS_TEST=True` right?